kitsune/products/migrations/0015_auto_20240722_0449.py

```diff
@@ -0,0 +1,527 @@
+        "slug": "settings",
+        "parent_topic_slug": "",
+    },
+    {
+        "old_title": "Customize settings and prefences",
+        "old_slug": "customize-settings-and-prefences-3",
+        "title": "Settings",
+        "slug": "settings",
+        "parent_topic_slug": "",
+    },
+    {
+        "old_title": "Customize settings and prefences",
+        "old_slug": "customize-settings-and-prefences-4",
+        "title": "Settings",
+        "slug": "settings",
+        "parent_topic_slug": "",
+    },
+    {
+        "old_title": "Install and manage add-ons",
+        "old_slug": "install-and-manage-add-ons-3",
+        "title": "Add-ons, extensions, and themes",
+        "slug": "add-ons-extensions-and-themes",
+        "parent_topic_slug": "settings",
+    },
+    {
+        "old_title": "Privacy and security settings",
+        "old_slug": "privacy-and-security-settings",
+        "title": "Privacy and security",
+        "slug": "privacy-and-security",
+        "parent_topic_slug": "",
+    },
+    {
+        "old_title": "Passwords, forms and search",
+        "old_slug": "passwords-forms-and-search",
+        "title": "Passwords and sign in",
+        "slug": "passwords-and-sign-in",
+        "parent_topic_slug": "",
+    },
+    {
+        "old_title": "Customize settings and prefences",
+        "old_slug": "customize-settings-and-prefences-5",
+        "title": "Customization",
+        "slug": "customization",
+        "parent_topic_slug": "settings",
+    },
+    {
+        "old_title": "Thunderbird controls and buttons",
+        "old_slug": "thunderbird-controls-and-buttons",
+        "title": "Settings",
+        "slug": "settings",
+        "parent_topic_slug": "",
+    },
+    {
+        "old_title": "Calendar",
+        "old_slug": "calendar",
+        "title": "Calendar",
+        "slug": "calendar",
+        "parent_topic_slug": "email-and-messaging",
+    },
+    {
+        "old_title": "Install and update",
+        "old_slug": "install-and-update-2",
+        "title": "Installation and updates",
+        "slug": "installation-and-updates",
+        "parent_topic_slug": "",
+    },
+    {
+        "old_title": "Thunderbird versions and languages",
+        "old_slug": "thunderbird-versions-and-languages",
+        "title": "Settings",
+        "slug": "settings",
+        "parent_topic_slug": "",
+    },
+    {
+        "old_title": "Copy your personal information from one Thunderbird to another",
+        "old_slug": "copy-your-personal-information-from-one-thunderbird-to-another",
+        "title": "Import and export email",
+        "slug": "import-and-export-email",
+        "parent_topic_slug": "email-and-messaging",
+    },
+    {
+        "old_title": "Emails",
+        "old_slug": "emails",
+        "title": "Email and messaging",
+        "slug": "email-and-messaging",
+        "parent_topic_slug": "",
+    },
+    {
+        "old_title": "Set up email",
+        "old_slug": "set-up-email",
```

Comment:
I think this should be `set-up-email-thunderbird` instead. There are no topics with the slug `set-up-email` in stage or prod.